# HG changeset patch
# User José Luis García Pallero <jgpallero@gmail.com>
# Date 1428975430 25200
#      Mon Apr 13 18:37:10 2015 -0700
# Node ID ad1a2c03992bb1a95f309139da540f2b8d4dcabf
# Parent  3b90259f17f32083f44d97a191512735cc0c9106
es_ES.ts: Update Spanish translation.

* es_ES.ts: Update Spanish translation.

diff --git a/libgui/languages/es_ES.ts b/libgui/languages/es_ES.ts
--- a/libgui/languages/es_ES.ts
+++ b/libgui/languages/es_ES.ts
@@ -99,27 +99,27 @@
     <message>
         <location filename="../graphics/Figure.cc" line="+282"/>
         <source>Figure ToolBar</source>
         <translation>Barra de herramientas de figura</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>Axes</source>
-        <translation type="unfinished"></translation>
+        <translation>Ejes</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Grid</source>
-        <translation type="unfinished"></translation>
+        <translation>Malla</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Autoscale</source>
-        <translation type="unfinished"></translation>
+        <translation>Autoescalado</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>&amp;File</source>
         <translation>&amp;Archivo</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -304,52 +304,58 @@
         <location line="-2"/>
         <source>Pre&amp;vious Bookmark</source>
         <translation>Marcador &amp;anterior</translation>
     </message>
     <message>
         <location line="-1237"/>
         <source>New function name:
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Nombre para la nueva función:
+</translation>
     </message>
     <message>
         <location line="+99"/>
         <source>Could not start custom file editor
 %1</source>
-        <translation type="unfinished"></translation>
+        <translation>No se puede abrir el editor de texto personalizado
+%1</translation>
     </message>
     <message>
         <location line="+125"/>
         <source>Could not open file
 %1
 for read: %2.</source>
-        <translation type="unfinished"></translation>
+        <translation>El fichero
+%1
+no puede ser abierto para lectura: %2.</translation>
     </message>
     <message>
         <location line="+19"/>
         <source>File
 %1
 does not exist. Do you want to create it?</source>
-        <translation type="unfinished"></translation>
+        <translation>El fichero
+%1
+no existe. ¿Desea crearlo?</translation>
     </message>
     <message>
         <location line="+998"/>
         <source>Toggle &amp;Bookmark</source>
         <translation>&amp;Alternar marcadores</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Remove All Bookmarks</source>
         <translation>&amp;Eliminar todos los marcadores</translation>
     </message>
     <message>
         <location line="+20"/>
         <source>Show &amp;Whitespace Characters</source>
-        <translation type="unfinished"></translation>
+        <translation>Mostrar &amp;espacios en blanco</translation>
     </message>
     <message>
         <location line="+34"/>
         <source>&amp;Next Breakpoint</source>
         <translation>&amp;Siguiente punto de interrupción</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -411,24 +417,28 @@ does not exist. Do you want to create it
         <source>Cancel</source>
         <translation>Cancelar</translation>
     </message>
     <message>
         <location line="+20"/>
         <source>Could not open file
 %1
 for write: %2.</source>
-        <translation type="unfinished"></translation>
+        <translation>El fichero
+%2
+no se puede abrir para escritura: %2.</translation>
     </message>
     <message>
         <location line="+62"/>
         <source>File not saved! A file with the selected name
 %1
 is already open in the editor</source>
-        <translation type="unfinished"></translation>
+        <translation>¡Fichero no guardado! Ya hay un fichero con el nombre seleccionado
+%1
+abierto en el editor</translation>
     </message>
     <message>
         <location line="+762"/>
         <source>&amp;Recent Editor Files</source>
         <translation>Archivos &amp;recientes</translation>
     </message>
     <message>
         <location line="+8"/>
@@ -543,17 +553,17 @@ is already open in the editor</source>
     <message>
         <location line="+3"/>
         <source>Navi&amp;gation</source>
         <translation>&amp;Navegación</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Move to Matching Brace</source>
-        <translatorcomment>Las posibilidades son aréntesis, corchete o llave.</translatorcomment>
+        <translatorcomment>Las posibilidades son paréntesis, corchete o llave.</translatorcomment>
         <translation>Mover al paréntesis/corchete/llave correspondiente</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Select to Matching Brace</source>
         <translation>Seleccionar hasta el paréntesis/corchete/llave correspondiente</translation>
     </message>
     <message>
@@ -579,22 +589,22 @@ is already open in the editor</source>
     <message>
         <location line="+3"/>
         <source>Show &amp;Line Numbers</source>
         <translation>Mostrar números de &amp;línea</translation>
     </message>
     <message>
         <location line="+128"/>
         <source>Editor</source>
-        <translation type="unfinished">Editor</translation>
+        <translation>Editor</translation>
     </message>
     <message>
         <location line="-120"/>
         <source>Show Line &amp;Endings</source>
-        <translation>Mostrar &amp;fin de líneas</translation>
+        <translation>Mostrar &amp;fines de línea</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Show &amp;Indentation Guides</source>
         <translation>Mostrar guías de &amp;sangría</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -634,17 +644,17 @@ is already open in the editor</source>
     <message>
         <location line="+2"/>
         <source>&amp;Documentation on Keyword</source>
         <translation>&amp;Documentación de palabras clave</translation>
     </message>
     <message>
         <location line="-969"/>
         <source>The associated file editor tab has disappeared.</source>
-        <translation>La pestaña para la edición del archivo ha desaparecido.</translation>
+        <translation>La pestaña del editor de texto asociado ha desaparecido.</translation>
     </message>
     <message>
         <location line="+731"/>
         <source>&amp;File</source>
         <translation>&amp;Archivo</translation>
     </message>
     <message>
         <location line="+36"/>
@@ -672,17 +682,17 @@ is already open in the editor</source>
     <message>
         <location line="+10"/>
         <source>line:</source>
         <translation>Línea:</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>col:</source>
-        <translation>Col:</translation>
+        <translation>Columna:</translation>
     </message>
     <message>
         <location line="+156"/>
         <source>%1 is a built-in function</source>
         <translation>%1 es una función integrada en Octave</translation>
     </message>
     <message>
         <location line="+31"/>
@@ -707,48 +717,63 @@ is already open in the editor</source>
     </message>
     <message>
         <location line="+4"/>
         <source>The file
 
   %1
 
 is about to be closed but has been modified.  %2</source>
-        <translation type="unfinished"></translation>
+        <translation>El fichero
+
+  %1
+
+se va a cerrar, pero ha sido modificado.  %2</translation>
     </message>
     <message>
         <location line="+213"/>
         <source>Could not open file %1 for write:
 %2.</source>
-        <translation type="unfinished"></translation>
+        <translation>El fichero %1 no se puede abrir para escritura:
+%2.</translation>
     </message>
     <message>
         <location line="+177"/>
         <source>&quot;%1&quot;
 is not a valid identifier.
 
 If you keep this file name, you will not be able to
 call your script using its name as an Octave command.
 
 Do you want to choose another name?</source>
-        <translation type="unfinished"></translation>
+        <translation>&quot;%1&quot;
+no es un identificador válido.
+
+Si mantiene este nombre para el fichero, no podrá llamar al script
+usando su nombre como un comando de Octave.
+
+¿Desea seleccionar otro nombre?</translation>
     </message>
     <message>
         <location line="+101"/>
         <source>
 
 Warning: The contents in the editor is modified!</source>
-        <translation type="unfinished"></translation>
+        <translation>
+
+Aviso: ¡el contenido del editor ha sido modificado!</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>It seems that the file
 %1
 has been deleted or renamed. Do you want to save it now?%2</source>
-        <translation type="unfinished"></translation>
+        <translation>Parece que el fichero
+%1
+ha sido eliminado o renombrado. ¿Desea salvarlo ahora?%2</translation>
     </message>
     <message>
         <location line="-210"/>
         <source>Line Endings:</source>
         <translation>Fin de línea:</translation>
     </message>
     <message>
         <location line="+23"/>
@@ -796,17 +821,17 @@ has been deleted or renamed. Do you want
     <message>
         <location line="+18"/>
         <source>Enter the path or filename</source>
         <translation>Introduzca la dirección o el nombre del archivo</translation>
     </message>
     <message>
         <location line="+24"/>
         <source>Set Octave directory to current browser directory</source>
-        <translation type="unfinished"></translation>
+        <translation>Asignar el directorio actual del navegador como directorio de Octave</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Actions on current directory</source>
         <translation>Acciones en el directorio actual</translation>
     </message>
     <message>
         <location line="+15"/>
@@ -818,29 +843,32 @@ has been deleted or renamed. Do you want
         <location line="+388"/>
         <source>Find Files...</source>
         <translation>Buscar archivos...</translation>
     </message>
     <message>
         <location line="+123"/>
         <source>Rename file/directory:
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Renombrar fichero/directorio
+</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>
  to: </source>
-        <translation type="unfinished"></translation>
+        <translation>
+ a: </translation>
     </message>
     <message>
         <location line="+26"/>
         <source>Are you sure you want to delete
 </source>
-        <translation type="unfinished"></translation>
+        <translation>¿Está seguro de que quiere borrar
+</translation>
     </message>
     <message>
         <location line="+214"/>
         <source>New Directory</source>
         <translation>Nuevo directorio</translation>
     </message>
     <message>
         <location line="-784"/>
@@ -877,17 +905,17 @@ has been deleted or renamed. Do you want
         <location line="-394"/>
         <location line="+397"/>
         <source>New Directory...</source>
         <translation>Nuevo directorio...</translation>
     </message>
     <message>
         <location line="-335"/>
         <source>Activate to open in editor, right click for alternatives</source>
-        <translation>Activar para abrir en el editor, haga clic derecho para ver las alternativas</translation>
+        <translation>Activar para abrir en el editor. Haga clic derecho para ver las alternativas</translation>
     </message>
     <message>
         <location line="+236"/>
         <source>File size</source>
         <translation>Tamaño del archivo</translation>
     </message>
     <message>
         <location line="+8"/>
@@ -1021,17 +1049,21 @@ has been deleted or renamed. Do you want
     </message>
     <message>
         <location line="+7"/>
         <source>&lt;html&gt;&lt;body&gt;
 &lt;p&gt;We hope you find Octave to be a useful tool.&lt;/p&gt;
 &lt;p&gt;If you encounter problems, there are a number of ways to get help, including commercial support options, a mailing list, a wiki, and other community-based support channels.
 You can find more information about each of these by visiting &lt;a href=&quot;http://octave.org/support.html&quot;&gt;http://octave.org/support.html&lt;/a&gt; (opens in external browser).&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;body&gt;
+&lt;p&gt;Esperamos que Octave le sea útil.&lt;/p&gt;
+&lt;p&gt;Si se encuentra con problemas, existen varias vías para obtener ayuda, las cuales incluyen soporte comercial, listas de correo, un wiki y otros canales de comunicación soportados por la comunidad.
+Puede encontrar más información visitando &lt;a href=&quot;http://octave.org/support.html&quot;&gt;http://octave.org/support.html&lt;/a&gt; (se abre en una ventana externa del navegador).&lt;/p&gt;
+&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>&lt;html&gt;&lt;head&gt;
 &lt;style&gt;
 a:link { text-decoration: underline; color: #0000ff; }
 &lt;/style&gt;
 &lt;head/&gt;&lt;body&gt;
@@ -1044,19 +1076,19 @@ a:link { text-decoration: underline; col
 &lt;/body&gt;&lt;/html&gt;</source>
         <translation>&lt;html&gt;&lt;head&gt;
 &lt;style&gt;
 a:link { text-decoration: underline; color: #0000ff; }
 &lt;/style&gt;
 &lt;head/&gt;&lt;body&gt;
 &lt;p&gt;Para más información acerca de Octave:&lt;/p&gt;
 &lt;ul&gt;
-&lt;li&gt;Visite &lt;a href=&quot;http://octave.org&quot;&gt;http://octave.org&lt;/a&gt; (abre en un navegador externo)&lt;/li&gt;
-&lt;li&gt;Obtenga la documentación en linea en formato &lt;a href=&quot;http://www.gnu.org/software/octave/doc/interpreter/index.html&quot;&gt;html&lt;/a&gt; o como &lt;a href=&quot;http://www.gnu.org/software/octave/octave.pdf&quot;&gt;documento&lt;/span&gt;&lt;/a&gt; pdf(abre en un navegador externo)&lt;/li&gt;
-&lt;li&gt;Abra el navegador de la documentación de la GUI de Octave con el menú de ayuda&lt;/li&gt;
+&lt;li&gt;Visite &lt;a href=&quot;http://octave.org&quot;&gt;http://octave.org&lt;/a&gt; (se abre en una ventana externa del navegador)&lt;/li&gt;
+&lt;li&gt;Obtenga la documentación en linea en formato &lt;a href=&quot;http://www.gnu.org/software/octave/doc/interpreter/index.html&quot;&gt;html&lt;/a&gt; o como &lt;a href=&quot;http://www.gnu.org/software/octave/octave.pdf&quot;&gt;documento&lt;/span&gt;&lt;/a&gt; pdf(se abre en una ventana externa del navegador)&lt;/li&gt;
+&lt;li&gt;Abra el navegador de documentación de la GUI de Octave desde el menú de ayuda&lt;/li&gt;
 &lt;/ul&gt;
 &lt;/body&gt;&lt;/html&gt;
 
 
 </translation>
     </message>
 </context>
 <context>
@@ -1364,17 +1396,17 @@ a:link { text-decoration: underline; col
         <location line="+1"/>
         <source>Evaluate</source>
         <translation>Evaluar</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Create script</source>
         <translatorcomment>Ha sido utilizada la traducción &quot;guión&quot; de script: http://es.wikipedia.org/wiki/Script#Traducci.C3.B3n</translatorcomment>
-        <translation>Crear un guión</translation>
+        <translation>Crear un guión (script)</translation>
     </message>
 </context>
 <context>
     <name>initial_page</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="-179"/>
         <source>Welcome to Octave!</source>
         <translation>¡Bienvenido a Octave!</translation>
@@ -1469,17 +1501,17 @@ Pulse &apos;Siguiente&apos; para crear un archivo de configuración e iniciar Octave.&lt;/p&gt;
     <message>
         <location line="+134"/>
         <source>The release notes file &apos;%1&apos; is empty.</source>
         <translation>El archivo de notas de versión %1 está vacío.</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>The release notes file &apos;%1&apos; cannot be read.</source>
-        <translation>El archivo de notas de versión %1 no puede ser leído.</translation>
+        <translation>El archivo de notas de versión %1 no se puede leer.</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Octave Release Notes</source>
         <translation>Notas de la versión de Octave</translation>
     </message>
     <message>
         <location line="+192"/>
@@ -1639,17 +1671,17 @@ Pulse &apos;Siguiente&apos; para crear un archivo de configuración e iniciar Octave.&lt;/p&gt;
     <message>
         <location line="+3"/>
         <source>Community News</source>
         <translation>Noticias de la comunidad</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Toolbar</source>
-        <translation type="unfinished"></translation>
+        <translation>Barra de herramientas</translation>
     </message>
     <message>
         <location line="+18"/>
         <source>Enter directory name</source>
         <translation>Introduzca el nombre del directorio</translation>
     </message>
     <message>
         <location line="+9"/>
@@ -1848,17 +1880,19 @@ o active la conexión a la web para ver las novedades en el diálogo de configuración de red de Octave
         <source>Create</source>
         <translation>Crear</translation>
     </message>
     <message>
         <location line="-10"/>
         <source>File
 %1
 does not exist. Do you want to create it?</source>
-        <translation type="unfinished"></translation>
+        <translation>El fichero
+%1
+no existe. ¿Desea crearlo?</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Octave Editor</source>
         <translation>Editor de Octave</translation>
     </message>
     <message>
         <location line="+211"/>
@@ -1907,17 +1941,23 @@ does not exist. Do you want to create it
         <location filename="../src/resource-manager.cc" line="+245"/>
         <source>The settings file
 %1
 does not exist and can not be created.
 Make sure you have read and write permissions to
 %2
 
 Octave GUI must be closed now.</source>
-        <translation type="unfinished"></translation>
+        <translation>El fichero de configuración
+%1
+no existe y no puede ser creado.
+Asegúrese de que tiene permisos de lectura y escritura en
+%2
+
+La interfaz gráfica de Octave se cerrará.</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Octave Critical Error</source>
         <translation>Error crítico de Octave</translation>
     </message>
 </context>
 <context>
@@ -2152,17 +2192,17 @@ Octave GUI must be closed now.</source>
         <location filename="../src/ui-settings-dialog.h" line="+11"/>
         <source>Create nonexistent files without prompting</source>
         <translation>Crear archivos no existentes sin consultar</translation>
     </message>
     <message>
         <location line="-429"/>
         <location filename="../src/ui-settings-dialog.h" line="-28"/>
         <source>command line (%f=file, %l=line):</source>
-        <translation>línea de comando(%f=archivo, %l=linea):</translation>
+        <translation>línea de comando (%f=archivo, %l=linea):</translation>
     </message>
     <message>
         <location line="+521"/>
         <location filename="../src/ui-settings-dialog.h" line="+43"/>
         <source>Terminal</source>
         <translation>Terminal</translation>
     </message>
     <message>
@@ -2258,17 +2298,17 @@ Octave GUI must be closed now.</source>
         <location line="+2"/>
         <source>Active</source>
         <translation>Activo</translation>
     </message>
     <message>
         <location line="-152"/>
         <location filename="../src/ui-settings-dialog.h" line="-10"/>
         <source>Normal</source>
-        <translation type="unfinished"></translation>
+        <translation>Normal</translation>
     </message>
     <message>
         <location line="+191"/>
         <location filename="../src/ui-settings-dialog.h" line="+12"/>
         <source>Background inactive</source>
         <translation>Fondo inactivo</translation>
     </message>
     <message>
@@ -2298,23 +2338,23 @@ Octave GUI must be closed now.</source>
         <location line="+83"/>
         <source>Browse</source>
         <translation>Explorar</translation>
     </message>
     <message>
         <location line="-1322"/>
         <location filename="../src/ui-settings-dialog.h" line="-78"/>
         <source>After Column</source>
-        <translation>Despues de la columna</translation>
+        <translation>Después de la columna</translation>
     </message>
     <message>
         <location line="+144"/>
         <location filename="../src/ui-settings-dialog.h" line="+6"/>
         <source>Tab width min.</source>
-        <translation>Anchura mínima del tabulador.</translation>
+        <translation>Anchura mínima del tabulador</translation>
     </message>
     <message>
         <location line="+64"/>
         <location filename="../src/ui-settings-dialog.h" line="+1"/>
         <source>max.</source>
         <translation>máx.</translation>
     </message>
     <message>
@@ -2532,17 +2572,17 @@ Octave GUI must be closed now.</source>
         <location filename="../src/ui-settings-dialog.h" line="-18"/>
         <source>Select this option to prevent conflicts with readline shortcuts</source>
         <translation>Seleccione esta opción para evitar conflictos con los atajos de la línea de comandos</translation>
     </message>
     <message>
         <location line="+3"/>
         <location filename="../src/ui-settings-dialog.h" line="+2"/>
         <source>Disable global shortcuts when terminal window has focus</source>
-        <translation>Deshabilitar loa atajos globales cuando la ventana del terminal se enfoque </translation>
+        <translation>Deshabilitar los atajos globales cuando la ventana del terminal se enfoque </translation>
     </message>
     <message>
         <location line="+31"/>
         <location line="+28"/>
         <location filename="../src/ui-settings-dialog.h" line="+1"/>
         <location line="+4"/>
         <source>Use this set</source>
         <translation>Utilice este conjunto</translation>
@@ -3140,62 +3180,62 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+4"/>
         <source>Show Line Numbers</source>
         <translation>Mostrar numeros de línea</translation>
     </message>
     <message>
         <location line="-145"/>
         <source>Step</source>
-        <translation type="unfinished">Siguiente instrucción</translation>
+        <translation>Siguiente instrucción</translation>
     </message>
     <message>
         <location line="+36"/>
         <source>Reset Default Window Layout</source>
-        <translation type="unfinished">Restablecer el esquema de ventana predeterminado</translation>
+        <translation>Restablecer el esquema de ventana predeterminado</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Report Bug</source>
-        <translation type="unfinished">Informar de fallo</translation>
+        <translation>Informar de fallo</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>Close Other Files</source>
-        <translation type="unfinished">Cerrar otros archivos</translation>
+        <translation>Cerrar otros archivos</translation>
     </message>
     <message>
         <location line="+30"/>
         <source>Show Completion List</source>
-        <translation type="unfinished"></translation>
+        <translation>Mostrar lista de comandos coincidentes</translation>
     </message>
     <message>
         <location line="+24"/>
         <source>Convert Line Endings to Windows</source>
-        <translation type="unfinished"></translation>
+        <translation>Convertir fines de línea a Windows</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Convert Line Endings to Unix</source>
-        <translation type="unfinished"></translation>
+        <translation>Convertir fines de línea a Unix</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Convert Line Endings to Mac</source>
-        <translation type="unfinished"></translation>
+        <translation>Convertir fines de línea a Mac</translation>
     </message>
     <message>
         <location line="+25"/>
         <source>Show Whitespace Characters</source>
-        <translation type="unfinished"></translation>
+        <translation>Mostrar espacios en blanco</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Show Line Endings</source>
-        <translation>Mostrar fin de línea</translation>
+        <translation>Mostrar fines de línea</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Show Indentation Guides</source>
         <translation>Mostrar guías de sangría</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -3305,17 +3345,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+4"/>
         <source>Run</source>
         <translation>Ejecutar</translation>
     </message>
     <message>
         <location line="+124"/>
         <source>Enter new Shortcut for Set %1</source>
-        <translation>Introduzca nuevo atajo para conjunto %1</translation>
+        <translation>Introduzca nuevo atajo para el conjunto %1</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Apply the desired shortcut or click on the right button to reset the shortcut to its default.</source>
         <translation>Aplique el atajo que desee o haga clic en el botón derecho para restablecer el atajo a su valor predeterminado.</translation>
     </message>
     <message>
         <location line="+7"/>
